cmake/Modules/Findlibdbi.cmake

```diff
@@ -0,0 +1,13 @@
+# Determine compiler flags for libdbi
+# Once done this will define
+# LIBDBI_FOUND - libdbi found
+
+find_path(LIBDBI_INCLUDE_DIR NAMES dbi.h HINTS "${LIBDBI_DIR}/include/dbi")
+
+find_library(LIBPGSQL_DRIVER_LIBRARY NAMES dbdpgsql HINTS "${LIBDBI_DIR}/lib/${CMAKE_C_LIBRARY_ARCHITECTURE}/dbd")
+
+find_library(LIBDBI_DEV_LIBRARY NAMES dbi)
```

Comment:
I think it's being found because you are installing the libraries in a standard system directory (/usr ?) - if the library is installed in a non-standard location, it wouldn't be detected. Adding HINTS would fix that